test/unit/callbuilder_test.js

```diff
@@ -0,0 +1,169 @@
+describe("callbuilder.js tests", function () {
+  beforeEach(function () {
+
+  });
+
+  afterEach(function () {
+
+  });
```

Comment:
`beforeEach` and `afterEach` can be removed if not used.

Reply:
Done.